co2eq/flights/index.js

```diff
@@ -205,7 +205,13 @@ function getLoadFactors(activity) {
   if (activity.departureAirportCode && activity.destinationAirportCode) {
     const departureAirportRegion = airportIataCodeToRegion(activity.departureAirportCode);
     const destinationAirportRegion = airportIataCodeToRegion(activity.destinationAirportCode);
-    if (departureAirportRegion && destinationAirportRegion) {
+    if (
+      departureAirportRegion &&
+      destinationAirportRegion &&
```

Comment:
Good to check indeed.

At the time I did not know that everything was in place to add unit tests to carbon models. I would therefore suggest add a guideline in the `README.md` that modifications to the carbon models require the implementation of unit tests that specifically verify use cases related to that modification.

Reply:
Agreed with the idea, but what if we simply make sure unit tests are there when we review the PR? I'm not sure it requires a change to the README.
Are you ok with those changes though?